### PR TITLE
Add ProgressComponent to projects resumes profile

### PR DESCRIPTION
**Pull request by pedro-lima-jalafund**
[AB#392](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/392) [AB#444](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/444)

# Overview

Add your Azure DevOps story number above to link to devops
Add general description explaining what this PR achieves

- e.g. Adds resume creation page

## Outstanding Tasks

- [x] Create progress component
- [x] Add progress to projects
- [x] Add progress to resumes
- [x] Add progress to project profile
- [x] Add progress to resume profile

asdf

### CODE REVIEW

File: src/components/projects/Projects.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import Card from "./Card";
+import ProgressComponent from "../../common/ProgressComponent/ProgressComponent";
 import SearchIcon from "@material-ui/icons/Search";
 import { emptyImageSvg } from "../../constants";
```
Comment (lilian-delgado-jalafund) [inline src/components/projects/Projects.jsx]: You can delete it, since we are not using that image (emptyImageSvg)

**Review by lilian-delgado-jalafund — COMMENTED** [review]
Just that, then everything lgtm!

File: src/components/projects/Projects.jsx (hunk 1 above)
Comment (pedro-lima-jalafund) [reply src/components/projects/Projects.jsx]: done, thanks

**Review by lilian-delgado-jalafund — APPROVED** [review]

**Review by freddy-perez-jalafund — APPROVED** [review]
Appr0v3d 👍

**Review by jorge-lema-jalafund — APPROVED** [review]
approved

**Pull request merged**